packages/lab/src/dialog/DialogContext.tsx

```diff
@@ -2,9 +2,12 @@ import { createContext, useContext } from "react";
 import { ValidationStatus } from "@salt-ds/core";
 
 export const DialogContext = createContext<{
-  dialogId?: string;
-  status?: ValidationStatus;
-}>({});
+  dialogId?: string | undefined;
+  status?: ValidationStatus | undefined;
```

Comment:
```suggestion
  dialogId?: string;
  status?: ValidationStatus;
```
they mean the same so could omit